repository-data/webfiles/src/main/resources/site/src/scss/nhsd-intranet.scss

```diff
@@ -1,10 +1,12 @@
 // The Intranet styling system based on the NHSD styling system!
 @import "nhsuk";
```

Comment:
I see you are still importing the main website's CSS. Long-term these should be 100% isolated projects (i.e. CSS/JS for website and separate CSS/JS for intranet).  Then the common CSS/JS, used by both the website and the interment, comes in from the NHSD UI Toolkit.